tests/phpunit/tests/db.php

```diff
@@ -1454,6 +1454,24 @@ public function test_charset_switched_to_utf8mb4() {
 		$this->assertSame( 'utf8mb4', $result['charset'] );
 	}
 
+	/**
+	 * @ticket 58871
```

Comment:
Let's add an `@covers` annotation here.

While this test also calls `$wpdb::has_cap()`, the actual method under test here is `$wpdb::determine_charset()`, so we can have this:

```php
/**
 * @ticket 58871
 *
 * @covers wpdb::determine_charset
 */
```

Reply:
done. Thank you.